src/main/java/org/jenkinsci/plugins/scriptsecurity/sandbox/whitelists/StaticWhitelist.java

```diff
@@ -192,24 +194,31 @@ private static String printArgumentTypes(Class<?>[] parameterTypes) {
     }
 
     private static final Set<String> BLACKLIST;
-    static {
+
+    @SuppressFBWarnings(value = "OS_OPEN_STREAM", justification = "https://sourceforge.net/p/findbugs/bugs/786/")
```

Comment:
False positive (reported by @jglick :smile:  ). Moved to method to make the annotation happier.